### DISCUSSION TITLE
Unable to send support message using built-in contact method

### DISCUSSION THREAD

**Discussion by win7guru** [community/community#146781] [Mobile]
I was trying to send a question using the help contact option in the Github app.  However, every time I try tapping the send icon, it says, "Something went wrong."  Is anybody else having this issue?

**Comment by taki-on** [comment]
@win7guru it works fine for me in the iOS app. Does the issue happen in the Android app?

**Reply by win7guru** [reply]
Yes it does

**Reply by taki-on** [reply]
I will ping the Android team to have a look!

**Reply by win7guru** [reply]
Thank you

**Comment by Ownerbusinessjjfm** [comment]
They are using family link, then phone clone then apps clone, then Wi-fi for auto sync for wi-fi data...its happen to me 7 8 years... Now I knw
